deployability/modules/workflow_engine/examples/dtt1-agents-poc-aws.yaml

```diff
@@ -0,0 +1,106 @@
+    #- linux-ubuntu-20.04-amd64
+    #- linux-centos-8-amd64
+    - linux-redhat-7-amd64
+  infra-provider: aws
+  working-dir: /tmp/dtt1-poc
+
+tasks:
+  # Unique manager allocate task
+  - task: "allocate-manager-{manager}"
+    description: "Allocate resources for the manager."
+    do:
+      this: process
+      with:
+        path: python3
+        args:
+          - modules/allocation/main.py
+          - action: create
+          - provider: "{infra-provider}"
+          - size: large
+          - composite-name: "{manager}"
+          - inventory-output: "{working-dir}/manager-{manager}/inventory.yaml"
+          - track-output: "{working-dir}/manager-{manager}/track.yaml"
+          - label-termination-date: "1d"
+          - label-team: "QA"
+    foreach:
+      - variable: manager-os
+        as: manager
+
+  # Unique agent allocate task
+  - task: "allocate-agent-{agent}"
+    description: "Allocate resources for the agent."
+    do:
+      this: process
+      with:
+        path: python3
+        args:
+          - modules/allocation/main.py
+          - action: create
+          - provider: "{infra-provider}"
+          - size: large
+          - composite-name: "{agent}"
+          - inventory-output: "{working-dir}/agent-{agent}/inventory.yaml"
+          - track-output: "{working-dir}/agent-{agent}/track.yaml"
+          - label-termination-date: "1d"
+          - label-team: "QA"
+    foreach:
+      - variable: agent-os
+        as: agent
+
+  # Generic agent test task
+  - task: "run-agent-tests"
+    description: "Run tests install for the agent."
+    do:
+      this: process
+      with:
+        path: python3
+        args:
+          - modules/testing/main.py
+          - targets:
+            - wazuh-1: "{working-dir}/manager-linux-redhat-7-amd64/inventory.yaml"
+            - agent-1: "{working-dir}/agent-linux-ubuntu-20.04-amd64/inventory.yaml"
+            - agent-2: "{working-dir}/agent-linux-ubuntu-22.04-amd64/inventory.yaml"
+            - agent-3: "{working-dir}/agent-linux-oracle-9-amd64/inventory.yaml"
+            - agent-4: "{working-dir}/agent-linux-amazon-2-amd64/inventory.yaml"
+            #- agent-5: "{working-dir}/agent-linux-redhat-7-amd64/inventory.yaml"
+            - agent-5: "{working-dir}/agent-linux-redhat-8-amd64/inventory.yaml"
+            - agent-6: "{working-dir}/agent-linux-redhat-9-amd64/inventory.yaml"
+            - agent-7: "{working-dir}/agent-linux-centos-7-amd64/inventory.yaml"
+            - agent-8: "{working-dir}/agent-linux-centos-8-amd64/inventory.yaml"
+            - agent-9: "{working-dir}/agent-linux-debian-10-amd64/inventory.yaml"
+            - agent-10: "{working-dir}/agent-linux-debian-11-amd64/inventory.yaml"
+            - agent-11: "{working-dir}/agent-linux-debian-12-amd64/inventory.yaml"
+            #- agent-12: "{working-dir}/agent-linux-opensuse-15-amd64/inventory.yaml"
+            - agent-12: "{working-dir}/agent-linux-suse-15-amd64/inventory.yaml"
```

Comment:
Same, comments

Reply:
Done in [6311d92](https://github.com/wazuh/wazuh-qa/pull/5190/commits/6311d92724994e505494d41ca00dd7d9cddda76f)